projects/aca-playwright-shared/src/page-objects/components/viewer.component.ts

```diff
@@ -30,17 +30,28 @@ import { timeouts } from '../../utils';
 export class ViewerComponent extends BaseComponent {
   private static rootElement = 'adf-viewer';
 
-  private viewerLocator = this.getChild('.adf-viewer-render-layout-content');
+  public viewerLocator = this.getChild('.adf-viewer-render-layout-content');
   public closeButtonLocator = this.getChild('.adf-viewer-close-button');
   public fileTitleButtonLocator = this.getChild('.adf-viewer__file-title');
+  public pdfViewerContentPages = this.getChild('.adf-pdf-viewer__content .page');
 
   toolbar = new AcaHeader(this.page);
 
   constructor(page: Page) {
     super(page, ViewerComponent.rootElement);
   }
 
+  async isPdfViewerContentDisplayed(): Promise<boolean> {
+    const count = await this.pdfViewerContentPages.count();
+    return count > 0;
+  }
+
+  async waitForViewerToOpen(): Promise<void> {
+    await this.viewerLocator.waitFor({ state: 'visible', timeout: timeouts.medium });
+  }
+
   async isViewerOpened(): Promise<boolean> {
+    await this.viewerLocator.waitFor({ state: 'visible', timeout: timeouts.large });
```

Comment:
you already have "waitForViewerToOpen" method, consider calling that one

Reply:
Yes, call the same method corrected